### ISSUE TITLE
Go to next error in the file

### ISSUE THREAD

**Issue by fandy** [Microsoft/TypeScript-Sublime-Plugin#656]
It would be great if I can do `cmd+e` to go to the next error in the file. This is the only feature missing for me to transition completely from VScode.

**Comment by ozra** [comment]
Yeah, I have alt-up/down for prev/next error in all my editors, so now that I'm back in sublime because of better TS-support I go "doh!" all the time.

**Comment by ozra** [comment]
Oh, and also prev/next error in project! Essential!

**Comment by ibc** [comment]
Any update on this? It's really hard to navigate to next TS error (basically because it must be done manually).
